### DISCUSSION TITLE
modifying /tmp directory location to custom

### DISCUSSION THREAD

**Discussion by ricky1146** [FreshRSS/FreshRSS#6128] [Q&A] [answered]
Hello, I have a customized webserver, due to security reasons, so my /tmp folder is actually in a different location via the php.ini file.

an example /var/www/html/tmp/
then fresh rss location
/var/www/html/cms/freshrss

I understand the default location the installation gui looks for is /tmp/

I did read this post about the location on how to change
https://github.com/FreshRSS/FreshRSS/issues/1705

I followed  to the best i could on changing the /tmp folder location, as well as changing ownership to www-data.

I do have mpm-itk installed which i also gave the user permissions there also. But regardless  the gui doesn't recognize any change, even if apache2 is restarted, or reloaded. I am sure i am missing one step.

**Comment by Alkarex** [comment, marked as answer]
Hello,
We are getting the temporary path by [`sys_get_temp_dir()`](https://php.net/sys-get-temp-dir)

If this is not good enough, you can make a `./constants.local.php` file with:

```php
<?php
define('TMP_PATH', '/var/www/html/tmp/');
```

**Reply by ricky1146** [reply]
Great thanks, I am always missing one step, or adding an extra space or accidental line where there isn't one. Great support
